ecosystem/sep-0031.md

```diff
@@ -200,9 +200,9 @@ Name | Type | Description
 `amount` | number | Amount of payment in destination currency
 `asset_code` | string | Code of the asset the sending anchor intends to send. This must match one of the entries listed in the receiving anchor's `/info` endpoint.
 `asset_issuer` | string | (optional) The issuer of the stellar asset the sending anchor intends to send. If not specified, the asset sent must be issued by the receiving anchor.
-`sender_id` | string | The ID included in the [SEP-12](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#customer-put) `PUT /customer` response for the sending client.
-`receiver_id` | string | The ID included in the [SEP-12](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#customer-put) `PUT /customer` response for the receiving client.
-`fields` | object | A key-pair object containing the values requested by the receiving anchor in their `/info` endpoint, broken out by sender, receiver, and transacton.
+`sender_id` | `string` | (optional) The ID included in the [SEP-12](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#customer-put) `PUT /customer` response for the sending client.
```

Comment:
lets make it clear this is not optional if there's a sender_sep12_type in the info endpoint